script/cargo.py

```diff
@@ -93,7 +93,8 @@ def build(rustup_home, cargo_home, manifest_path, build_path, target, is_debug):
     cargo_home = os.path.join(cargo_home, RUST_DEPS_PACKAGE_VERSION)
     env['CARGO_HOME'] = cargo_home
 
-    rustup_bin = os.path.join(rustup_home, 'bin')
+    rustup_bin = os.path.abspath(os.path.join(rustup_home, 'bin'))
```

Comment:
weird, that should already be an absolute path

Reply:
It was relative, I logged it.

Reply:
oh, right. rebase_path does a relative path